thefuck/rules/wrong_hyphen_before_subcommand.py

```diff
@@ -0,0 +1,22 @@
+from thefuck.utils import get_all_executables
+from thefuck.specific.sudo import sudo_support
+
+
+@sudo_support
+def match(command):
+    if '-' not in command.script_parts[0] or command.script_parts[0] in get_all_executables():
+        return False
+
+    cmd, subcmd = command.script_parts[0].split('-')[:2]
```

Comment:
Now that `subcmd` isn't necessary anymore, this line could be changed to:
```suggestion
    cmd = command.script_parts[0].split('-', 1)[0]
```